packages/hoppscotch-backend/src/admin/admin.service.ts

```diff
@@ -349,6 +432,59 @@ export class AdminService {
     return E.right(delUser.right);
   }
 
+  /**
+   * Remove user (not Admin) accounts by UIDs
+   * @param userUid User UIDs
+   * @returns an Either of boolean or error
+   */
+  async removeUserAccounts(userUIDs: string[]) {
+    const allUsers = await this.userService.findUsersByIds(userUIDs);
+    if (allUsers.length === 0) return E.left(USER_NOT_FOUND);
```

Comment:
I think we should throw a different error for multiple users since the current one is used in many locations when a user account cannot be found.